scripts/feature_tests/upgrade/common.sh

```diff
@@ -0,0 +1,432 @@
+#!/bin/bash
+
+export IMAGE_OS=${IMAGE_OS:-"Ubuntu"}
+export IMAGE_USERNAME=${IMAGE_USERNAME:-"metal3"}
+export KUBERNETES_VERSION="v1.18.0"
+export KUBERNETES_BINARIES_VERSION="v1.18.0"
+export UPGRADED_K8S_VERSION_1="v1.18.1"
+export UPGRADED_K8S_VERSION_2="v1.18.2"
+export UPGRADED_BINARY_VERSION="v1.18.1"
+export CLUSTER_APIENDPOINT_IP=${CLUSTER_APIENDPOINT_IP:-"192.168.111.249"}
+# The next lines to be fixed in the CI
+export CAPM3PATH="/home/ubuntu/go/src/github.com/metal3-io/cluster-api-provider-metal3"
```

Comment:
Shall it be fixed now?

Reply:
PR 357 shall be merged first so needs to be re-checked.

Reply:
fixed in PR 357 to
/home/${USER}/go/src/github.com/metal3-io/cluster-api-provider-metal3